pkg/controller/ingressstate/ingress_state_controller.go

```diff
@@ -310,7 +313,7 @@ func checkAddresses(ctx context.Context, addresses []corev1.EndpointAddress, che
 			}
 			messages := checkPod(ctx, address.TargetRef)
 			if len(messages) > 0 {
-				unhealthyPodCount += 1
+				unhealthyPodCount++
```

Comment:
Unrelated?

Reply:
my IDE underlines that which is just bothersome, I marked that as a cosmetic change in a separate commit, hopefully that's ok